dags/nct.py

```diff
@@ -1,12 +1,15 @@
 import datetime
+import airflow.operators.sensors
 from airflow.models import DAG
 import utils.helpers as helpers
+from operators.http_to_s3_transfer import HTTPToS3Transfer
 
 args = {
     'owner': 'airflow',
     'depends_on_past': False,
-    'start_date': datetime.datetime.utcnow(),
+    'start_date': datetime.datetime(2017, 1, 1),
```

Comment:
Just to make this clear, are we defaulting to 2017-01-01 for any newly created DAGs, right? At least for now, I mean.